packages/flutter_driver/lib/src/extension/_extension_web.dart

```diff
@@ -18,6 +18,11 @@ import 'dart:js_util' as js_util;
 ///
 ///  * [_extension_io.dart], which has the dart:io implementation
 void registerWebServiceExtension(Future<Map<String, dynamic>> Function(Map<String, String>) call) {
+  // Define the result variable because packages/flutter_driver/lib/src/driver/web_driver.dart
+  // checks for this value to become non-null when waiting for the result. If this value is
+  // undefined at the time of the check, WebDriver throws an exception.
+  context[r'$flutterDriverResult'] = null;
```

Comment:
Why not use the same pattern as in line 26 to set this value?

```
js_util.setProperty(html.window, r'$flutterDriverResult', null);
```

Reply:
This is using the same pattern as in line 31 :) I'm not sure what the difference is between window and context. I think in a window context in the browser (as opposed to a web worker, service worker, or node.js server) there is no difference.

Reply:
Ah, I think this is mixing js-interop styles, that's confusing!